tests/unit/lms/services/lti_grading/_v13_test.py

```diff
@@ -62,6 +62,37 @@ def test_read_bad_response_lti_result(self, svc, ltia_http_service, bad_response
 
         assert not svc.read_result(sentinel.user_id)
 
+    def test_get_score_maximum(self, svc, ltia_http_service):
+        ltia_http_service.request.return_value.json.return_value = [
+            {"scoreMaximum": sentinel.score_max, "id": svc.line_item_url},
+            {"scoreMaximum": 1, "id": sentinel.other_lineitem},
+        ]
+
+        score = svc.get_score_maximum(sentinel.resource_link_id)
+
+        ltia_http_service.request.assert_called_once_with(
+            "GET",
+            "http://example.com/lineitems",
+            scopes=svc.LTIA_SCOPES,
+            params={"resource_link_id": sentinel.resource_link_id},
+            headers={"Accept": "application/vnd.ims.lis.v2.lineitemcontainer+json"},
+        )
+        assert score == sentinel.score_max
+
+    def test_get_score_maximum_with_error(self, svc, ltia_http_service):
+        ltia_http_service.request.side_effect = ExternalRequestError(
+            response=Mock(status_code=500)
+        )
+
+        assert not svc.get_score_maximum(sentinel.resource_link_id)
```

Comment:
Should this use `assert ... is None`? Strictly speaking the result is a float and `0.0` is a legal, but falsey value.

Reply:
https://www.imsglobal.org/spec/lti-ags/v2p0#scoremaximum
>The maximum score for this line item. Maximum score MUST be a numeric non-null value, strictly greater than 0.

Not technically legal hehe, but I agree, the `is None` check is better here :+1: